final/src/main/java/ua/epam/spring/hometask/converter/impl/AirDateConverterImpl.java

```diff
@@ -0,0 +1,42 @@
+package ua.epam.spring.hometask.converter.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import ua.epam.spring.hometask.converter.AirDateConverter;
+import ua.epam.spring.hometask.converter.AuditoriumConverter;
+import ua.epam.spring.hometask.domain.AirDate;
+import ua.epam.spring.hometask.dto.AirDateDTO;
+
+@Component
+public class AirDateConverterImpl implements AirDateConverter {
+
+    @Autowired
+    private AuditoriumConverter auditoriumConverter;
```

Comment:
Using mentioned above approach you can work against Converter interface:
@Autowired
@Qualifier("auditoriumConverter")
private Converter converter;